### ISSUE TITLE
Picture day

### ISSUE THREAD

**Issue by Braunn318** [Braunn318/futures-journal-pro#10]
Picture or screen in the Day tab, can be only uploaded as a file. It should be possible to insert a picture from the clipboard, same as in the trade details.
